modules/images-other-jenkins-env-var.adoc

```diff
@@ -90,4 +90,26 @@ log file to persist when a fatal error occurs. The fatal error file is saved at
 |Setting this value overrides the image used for the default maven agent pod configuration. A related image stream tag named `jenkins-agent-maven` is in the project. This variable must be set before Jenkins starts the first time for it to have an effect.
 |Default Maven agent image in Jenkins server:
 `image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-maven:latest`
+
+|`AGENT_BASE_IMAGE`
+|Setting this value overrides the image used for the 'jnlp' container in the sample kubernetes plug-in PodTemplates
+provided with this image.  Otherwise, the image from the 'jenkins-agent-base:latest' ImageStreamTag in the 'openshift'
+namespace is used.
+|Default:
+`image-registry.openshift-image-registry.svc:5000/openshift/jenkins-agent-base:latest`
+
+|`JAVA_BUILDER_IMAGE`
+|Setting this value overrides the image used for the 'java-builder' container in the 'java-builder' sample kubernetes
```

Comment:
Please replace all instances of straight quotes (') with backticks (`) throughout.